specifyweb/frontend/js_src/lib/localization/interactions.ts

```diff
@@ -90,6 +90,9 @@ export const interactionsText = createDictionary({
     'uk-ua': 'Без препаратів',
     'de-ch': 'Ohne Präparate',
   },
+  continueWithoutPreparations: {
+    'en-us': 'Continue without preparations',
```

Comment:
this is too long for a button label, but not sure if there is a briefer way that is just as clear

Reply:
Agreed, but we decided as a team to go for better communication with 'Continue without preparations' in this case.